examples/end-to-end-session-based/02-End-to-end-session-based-with-Yoochoose-PyT.ipynb

```diff
@@ -112,10 +112,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from transformers4rec.torch.utils.examples_utils import visualize_response
</pre><p>Let's remove this cell. we dont need it, and it wont work bcs of the <code>output</code> name.  Once you  add <code>model.top_k=..</code>  we'll return topk item_ids and scores already.Pls  see my comment above. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>